CHANGELOG.md

```diff
@@ -7,7 +7,13 @@ recommendations of [keepachangelog.com](http://keepachangelog.com/).
 
 ### Breaking Changes
 
-- None
+- `PaperTrail.request do ... end` now returns whatever value is returned from the given block. For example:
+``` ruby
+raise PaperTrail.request(whodunnit: "Michael Bay") do
+  ApplicationError
+end
+# => ApplicationError has been raised!
+```
```

Comment:
As I said, please put this under "Added". I don't see it as a breaking change.

Also, please omit the code example. The single sentence description is sufficient.

Reply:
We're changing the return type from always nil to sometimes nil, are you sure that's not a breaking change?

Reply:
> We're changing the return type of a public method from always nil to sometimes nil, are you sure that's not a breaking change?

Yes, you're right (both correct and righteous) to question this :grin: I don't consider it a breaking change, because I don't think anyone is depending on the fact that it is returning nil. What do you think? Do you think people will be depending on the nil?